### PR TITLE
Solves issue94, correct few lines after static analysis, move patches…

### PR DESCRIPTION
**Pull request by eolivi**
… folder to cmake/external

### CODE REVIEW

**Comment by agramfort** [comment]
can you add a test for the new check conductivity ?

besides LGTM

**Comment by agramfort** [comment]
fine with me if CIs are happy as well as @papadop

File: OpenMEEG/libs/OpenMEEG/include/OMstd.h [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
+Project Name : OpenMEEG
```
Comment (eolivi) [inline OpenMEEG/libs/OpenMEEG/include/OMstd.h]: is the name OMstd.h ok for you ?

File: OpenMEEG/libs/OpenMEEG/src/sensors.cpp [hunk 2]
```diff
@@ -207,7 +207,7 @@ namespace OpenMEEG {
             if (hasOrientations())
                 outfile << m_orientations.getlin(i) << " ";
             // if it has weights (other than 1)
-            if (std::abs(m_weights.sum() - m_weights.size()) > 1.e3*std::numeric_limits<double>::epsilon()) {
+            if (almost_equal(m_weights.sum(), static_cast<double>(m_weights.size()))) {
                 outfile << m_weights(i) << std::endl;
```
Comment (eolivi) [inline OpenMEEG/libs/OpenMEEG/src/sensors.cpp]: bug here ! add "not"

**Comment by agramfort** [comment]
@eolivi appveyor is not happy but travis seems to have succeeded building the standalone version. How can I check?

**Comment by eolivi** [comment]
yes it looks good !
I've added a prefix ("PR-")to the package so that it is uploaded to the forge even when coming from a PR.

./bin/om_assemble [-option] [filepaths...]
option :
   -HeadMat, -HM, -hm :   
       Compute Head Matrix for Symmetric BEM (left-hand side of linear system).
             Arguments :
               geometry file (.geom)
               conductivity file (.cond)
...

**Comment by agramfort** [comment]
where is the .tar.gz ?

**Comment by eolivi** [comment]
> I've added a prefix ("PR-")to the package so that it is uploaded to the forge even when coming from a PR.

actually the upload doesn't work, because the password variable are only available on master.

So let us review and merge ?

**Comment by agramfort** [comment]
alright let's see

**Pull request merged**